plutus-wallet-api/ledger/Ledger/Scripts.hs

```diff
@@ -183,18 +179,18 @@ instance ToJSON Script where
 instance FromJSON Script where
     parseJSON = JSON.decodeSerialise
 
--- See Note [Normalized types in Scripts]
--- | Lift a Haskell value into the corresponding 'Script'. This allows you to create
--- 'Script's at runtime, whereas 'compileScript' allows you to do so at compile time.
-lifted :: Lift a => a -> Script
-lifted = fromCompiledCode . liftCode
-
 instance ToJSON Data where
     toJSON = JSON.String . JSON.encodeSerialise
 
 instance FromJSON Data where
     parseJSON = JSON.decodeSerialise
 
+mkValidatorScript :: CompiledCode (Data -> Data -> PendingTx -> Bool) -> ValidatorScript
+mkValidatorScript = ValidatorScript . fromCompiledCode
+
+unValidatorScript :: ValidatorScript -> Script
+unValidatorScript = getValidator
+
```

Comment:
"unValidatorScript": not the most elegant word I've ever seen, but I guess you're bound by convention.

Reply:
Yeah, I'm not wild about this either. I wanted something that looks like the record accessor without actually being it... apparently you can't export the accessor function without letting people construct it.